src/balloon/index.jsx

```diff
@@ -37,6 +37,20 @@ export default ConfigProvider.config(Balloon, {
             props = { onVisibleChange: newOnVisibleChange, ...others };
         }
 
+        if ('triggerType' in props) {
+            const triggerType = Array.isArray(props.triggerType)
+                ? [...props.triggerType]
+                : [props.triggerType];
+
+            if (triggerType.indexOf('focus') > -1) {
+                deprecated(
+                    'triggerType[focus]',
+                    'triggerType[hover, click]',
```

Comment:
can we change this to say that if `focus` is desired they should use the `Tooltip` component?

Reply:
And maybe this should belong with Popup or Overlay components since the issue is really with them

Reply:
> i
The warning will be "triggerType[focus] is deprecated at [Balloon], use triggerType[hover, click] instead of it. If focus is desired maybe you should use [Balloon.Tooltip]". Is that OK?

Reply:
Yes, thats good